geode-docs/developing/query_index/creating_hash_indexes.html.md.erb

```diff
@@ -23,14 +23,20 @@ limitations under the License.
 
 ## <a id="concept_5C7614F71F394C62ACA1BDC5684A7AC4__section_8A927DFB29364DA7856E7FE122FC1654" class="no-quick-link"></a>Why Create a HashIndex
 
-By creating a hash index, you can vastly improve the memory usage of your index. If you are doing equality based queries, your indexing footprint is significantly reduced. As an example, previously when you created an index that contained string fields, copies of the strings were included in the index. By using hash indexes, we ensure that indexed expressions are canonicalized and stored in the index as pointers to the objects lying in the region. In internal test cases we have seen improvements in the 30% range depending on the keys and data being used.
+By creating a hash index, you can improve the memory usage of your index. If you are doing equality based queries, your indexing footprint is significantly reduced. As an example, previously when you created an index that contained string fields, copies of the strings were included in the index. By using hash indexes, we ensure that indexed expressions are canonicalized and stored in the index as pointers to the objects lying in the region. In internal test cases we have seen improvements in the 30% range depending on the keys and data being used.
```

Comment:
Who is the "we" in this? I think it should be specified, rather than left vague. Perhaps "Using hash indexes ensures that indexed expressions are canonicalized and stored int he index as pointers to the objects lying in the region. Internal test cases have shown improvements in the 30% range, depending on the keys and data being used." (Side note: Is it correct/appropriate to reference "internal test cases" here?)